src/Microsoft.ML.PCA/PcaTrainer.cs

```diff
@@ -469,6 +470,29 @@ public void SaveAsText(TextWriter writer, RoleMappedSchema schema)
             }
         }
 
+        public IDataView GetSummaryDataView(RoleMappedSchema schema)
+        {
+            var bldr = new ArrayDataViewBuilder(Host);
+
+            bldr.AddColumn("MeanVector", NumberType.R4, _mean);
+            bldr.AddColumn("ProjectedMeanVector", NumberType.R4, _meanProjected);
+
+            ValueGetter<VBuffer<DvText>> getSlotNames =
```

Comment:
>getSlotNames [](start = 41, length = 12)

We probably don't need the slot names here since they don't give any additional information other than the slot index.

Reply:
Why did you end up defining slot names?

---
In reply to: [191581196](https://github.com/dotnet/machinelearning/pull/185#discussion_r191581196) [](ancestors = 191581196)